.kitchen.docker.yml

```diff
@@ -0,0 +1,36 @@
+---
+driver:
+  name: docker
+
+driver_config:
+  require_chef_omnibus: false
+  use_sudo: false
+  privileged: true
+  private_key: .kitchen/docker_id_rsa
+  public_key: .kitchen/docker_id_rsa.pub
+
+platforms:
+<% %w(11.10 11.18 12.0 12.4 12.6).each do |chef_version| %>
```

Comment:
given the speed of the builds, I'm inclined to think it'd be best only testing 11.18 and 12.6 now